### ISSUE TITLE
[Feature Request] Microsoft Teams Webhook Integration

### ISSUE THREAD

**Issue by NathanAlvarado** [wazuh/wazuh#6349]
Adding another webhook integration similar to the existing Slack and PagerDuty ones could be useful for Microsoft Teams.

**Comment by jesusjimsa** [comment]
Hello, @NathanAlvarado.

Thank you for the feature request. We'll save this issue to develop a Microsoft Teams integration in the future. You can follow the progress here.

In the meantime, you can use the information in this blog post we posted some time ago to create a custom integration: https://wazuh.com/blog/how-to-integrate-external-software-using-integrator/

You have information on how to get the Microsoft Teams webhook here: https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using

**Comment by juljeanpierre** [comment]
+1 to this request. MS-Teams integration would be fantastic

**Comment by acfranken** [comment]
Yes, we are also using MS-Teams, integration  with hook_url to Teams would be great.

**Comment by kbendl** [comment]
Teams integration would work for us. As much as I like Slack, our IT organization is all Windows (Teams fanbois), all the time. So, Teams integration would be helpful. +1

**Comment by AndreHelwig** [comment]
Looking forward to see native MS-Teams integration for Wazuh.

**Comment by ridwansyah15** [comment]
still waiting for this integrations, it would be perfect for my work environment.

**Comment by rpoulard** [comment]
Same here, this integrations could be really helpfull

**Comment by kocercan** [comment]
+1 for teams, I've tried everything to integrate Wazuh with Teams but none of them work.

**Comment by redanthrax** [comment]
I wrote a quick and dirty integration while we wait for an official one.

https://github.com/redanthrax/wazuh-teams-integration
